src/brsTypes/Double.ts

```diff
@@ -20,8 +20,8 @@ export class Double implements Numeric, Comparable, Boxable {
      * @param value the value to store in the BrightScript double, rounded to 64-bit (double)
      *              precision.
      */
-    constructor(value: number) {
-        this.value = value;
+    constructor(value: number | Long) {
+        this.value = Number(value);
```

Comment:
Is calling `Number(value)` safe on an instance of `Long`?  I see that `Long` has a `.toNumber` method, so perhaps something like this would be safer:

```suggestion
        this.value = value instanceof Long ? value.toNumber() : value;
```

Reply:
Will change.